google-cloud-storage/src/test/java/com/google/cloud/storage/conformance/retry/CtxFunctions.java

```diff
@@ -123,8 +107,12 @@
         (ctx, c) -> {
           BlobInfo blobInfo =
               BlobInfo.newBuilder(ctx.getState().getBucket().getName(), c.getObjectName()).build();
-          Blob resolvedBlob = ctx.getStorage().create(blobInfo);
-          return ctx.map(s -> s.with(resolvedBlob));
+          Blob resolvedBlob = ctx.getStorage().create(blobInfo, c.getHelloWorldUtf8Bytes());
```

Comment:
Not necessary right now, but could you add comments as to why each object is necessary? Right now I'm not sure why I would use one over the other.